partner-information.html

```diff
@@ -63,7 +63,17 @@ <h1>Partner Information</h1>
 
 				</div>
 			</div>
-			
+
+			<div class="row sponsor-information">
+				<div class="col-md-4">
+					<img src="../../hackhpi_assets/sap.png" class="img-responsive">
```

Comment:
I don't think `../../` works here, as we use `./` everywhere else. Can you confirm @jak-ing ?

Reply:
Sorry, apparently I missed that one.

Reply:
Fixed @FWirtz